doc/tsdoc/README.md

```diff
@@ -205,13 +205,13 @@ ___
 
 ### invalidateEvaluationCommands
```

Comment:
should not be part of the diff. the doc has been outdated for a while and if we really wanted to update it, we should do that in a dedicated commit